scripts/genappimage.sh

```diff
@@ -62,8 +62,17 @@ delete_blacklisted
 # AppDir complete. Now package it as an AppImage.
 ########################################################################
 
-# No need for a fancy script. AppRun can just be a symlink to nvim.
-ln -s usr/bin/nvim AppRun
+# Appimage set the ARGV0 environment variable. This causes problems in zsh.
+# To prevent this, we use wrapper script to unset ARGV0 as AppRun.
+# See https://github.com/AppImage/AppImageKit/issues/852
+#
+cat << 'EOF' > AppRun
+#!/bin/bash
+
+unset ARGV0
+exec "$(dirname "$(readlink  -f "${0}")")/usr/bin/nvim" ${@+"$@"}
```

Comment:
is `${@+"$@"}`  correct?  Looking at https://mywiki.wooledge.org/BashGuide/Parameters , I only see `${x:+y}`, not `${x+y}`. 

What is `${@+"$@"}` supposed to do?